routes/mylibrary.js

```diff
@@ -97,52 +100,99 @@ function addUserBook(userId, bookId, conditionId, callback)
 // @desc    Get current users mylibrary
 router.get('/', common.isAuthenticated, (req, res, next) => {
     const userId = req.session.userId;
-    var payload = {
-        title: 'My Library',
+    var payload = { 
+        title: 'My Library', 
         library: [],
-        titles: [],
-        conditions: [],
         avail: 0,
         rcvd: 0
     };
-    var selectAllBooks = `SELECT UserBooks.id AS userBookId, Books.id AS bookId, available AS swap, listingDate AS date, Books.title AS title, Books.imgUrl AS imgUrl
+    var selectAllBooks = `SELECT UserBooks.id AS userBookId, available AS swap, listingDate AS date, description, Books.author As author, Books.title AS title, Books.imgUrl AS imgUrl
                         FROM UserBooks
                         INNER JOIN Books ON Books.id = UserBooks.bookId
-                        WHERE UserBooks.userId = ?`
-    var selectAllConditions = 'SELECT id, description FROM Conditions ORDER BY id'
```

Comment:
By removing these two queries (selectAllConditions and selectAllTitles) you are breaking the Add Book modal which Kylee coded. These need to remain here with all the logic previously coded as the modal is a one time fill on page load.